src/core/agent/services/ipexCommunicationService.ts

```diff
@@ -95,7 +95,8 @@ class IpexCommunicationService extends AgentService {
     await this.admitIpex(
       notifRecord.a.d as string,
       holder.signifyName,
-      exn.exn.i
+      exn.exn.i,
+      [exn.exn.e.acdc.s, exn.exn.e.acdc?.e?.qvi?.s]
```

Comment:
@bao-sotatek I think you're not understanding what I mean. I want our wallet to be able to dynamically resolve all schemas associated with _any_ chained credential.

So we cannot assume that there will be exactly a `qvi` credential attached.

You need to:
- cover `exn.e.acdc.s` as you have,
- iterate over each edge (`exn.e.acdc.e`) and extract the schema from each of them.

This should be arbitrary for ANY credential our wallet receives.

Reply:
oh, so i will get all chained ACDC schemas in Edge and resolve it like this?: 
```
  const chainedSchemaSaids = Object.keys(exn.exn.e.acdc?.e || {}).map(
      (key) => exn.exn.e.acdc.e?.[key]?.s
    );

    await this.admitIpex(
      notifRecord.a.d as string,
      holder.signifyName,
      exn.exn.i,
      [exn.exn.e.acdc.s, ...chainedSchemaSaids]
    );
```

Reply:
Something like that, yes